images/macos/templates/macOS-13.arm64.anka.pkr.hcl

```diff
@@ -34,17 +34,17 @@ variable "github_api_pat" {
   default = ""
 }
 
-variable "xcode_install_user" {
+variable "xcversion_auth_cookie" {
```

Comment:
Why do we need this variable? it doesn't seem to be used anywhere

Reply:
done